lego/apps/lending/fixtures/development_lending_requests.yaml

```diff
@@ -0,0 +1,17 @@
+- fields:
+    lendable_object: 1
+    created_by: 3
+    start_date: '2024-02-02T23:16:00+00:00'
+    end_date: '2024-02-02T23:18:00+00:00'
+    message: "Jeg vil låne grill :)"
+    pending: True # Should be a status field with multiple options
```

Comment:
Wouldn't it be good to make this an enum from the start? To avoid having to make migrations later.